fontir/src/variations.rs

```diff
@@ -113,6 +121,75 @@ impl VariationModel {
     pub fn default_location(&self) -> &NormalizedLocation {
         &self.default
     }
+
+    /// Convert absolute positions at master locations to offsets.
+    ///
+    /// <ul>
+    ///     <li>All keys in points must be known to the variation model.</li>
+    ///     <li>All point vectors must have the same length.</li>
+    ///     <li>A point vector for the default location is required</li>
+    /// </ul>
+    ///
+    /// Note that it is NOT required to provide a point sequence for every location known
+    /// to the variation model.
+    ///
+    /// Returns a delta, in the form of a [Vec2], for every input point. Intended use is to support
+    /// construction of a variation store.
+    ///
+    /// Rust version of <https://github.com/fonttools/fonttools/blob/3b9a73ff8379ab49d3ce35aaaaf04b3a7d9d1655/Lib/fontTools/varLib/models.py#L449-L461>
+    pub fn deltas(
+        &self,
+        point_seqs: HashMap<NormalizedLocation, Vec<Point>>,
```

Comment:
maybe we could make this method generic. Right now this would work for Points only (hence, gvar), but the same method should be used for other single values that are not points or (x,y) tuples. The fonttools original doesn't mention "points" at all, only talks about "master values". When building a gvar, that method is fed GlyphCordinates which is equivalent to Vec<Point> and supports math operations like subtraction or multiplication by a scalar.

Reply:
Yes. In practice, fonttools' VariationModel works for any object that supports `__sub__`, `__add__` and `__mul__`. This is enormously useful and IMO a key point in Behdad's excellent design.

Reply:
I _think_ I can do the same but I didn't want to start there. Worst case I can give multiple methods, one for 2d and one for 1d ... hopefully fonts never use 3+.

Reply:
Context: am Rust noob

Reply:
std::ops sure looks like it'll work!

Reply:
Now with test.